refresh_versions.toml

```diff
@@ -0,0 +1,13 @@
+charm_major = 1
+workload = "16.8"
+
+charm = "16/1.3.0" # TODO remove
+
+[snap]
+name = "charmed-postgresql"
+
+[snap.revisions]
+# amd64
+x86_64 = "170"
+# arm64
+aarch64 = "169"
```

Comment:
Moving the snap revs will conflict with https://github.com/canonical/postgresql-operator/pull/881. Merging will need to be synchronised.

Reply:
ack, i will update once rebasing after 881 is merged